doc/hotfix-process.md

```diff
@@ -0,0 +1,54 @@
+Hotfix Release Process
+======================
+
+## Create a hotfix branch
+
+Crete a hotfix branch from the previous release tag, and push it to the main
+repository:
+
+    $ git branch hotfix-<RELEASE> <RELEASE_PREV>
```

Comment:
When I've done releases I get confused when a version-like string should begin with a `v…` prefix or not. For example `v1.2.3rc7` versus `1.2.3rc7`. Should `<RELEASE>` and `<RELEASE_PREV>` begin with a `v…` or not?

Reply:
Also, what is the versioning scheme for hotfix releases? I believe the version reported by `zcashd -version` should have a `-<N>` suffix, so should `<RELEASE>` be _exactly_ the same as `<RELEASE_PREV>` with `-1` appended, unless there is already a hotfix suffix in which case it should increment?

Please spell out these two details in this document, since I know if _I_ were doing a hotfix I'd be likely to forget these small but consequential details.

Reply:
By convention, our tags start with a `v`, so any time we reference a version in git we use a `v` as prefix.

I concur that hotfix versions should (only) increment the build number.